### DISCUSSION TITLE
part-ii/2016-83/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#317] [Question comments]
Part II 2016 Logic and Set Theory: Paper 2, Section II, F 

https://questions.tripos.org/part-ii/2016-83/

**Comment by ArthurConmy** [comment]
It's isomorphic to $V_\omega$. Every integer $n$ is a unique finite subset of $\omega$, namely the particular $m$ so that $\langle m, n \rangle \in r $. $V_\omega$ is exactly the set of finite subsets of $\omega$, done.
